src/ORM/EagerLoader.php

```diff
@@ -809,7 +809,7 @@ protected function _collectKeys(array $external, Query $query, $statement): arra
     protected function _groupKeys(BufferedStatement $statement, array $collectKeys): array
     {
         $keys = [];
-        while ($result = $statement->fetch('assoc')) {
+        foreach ($statement->fetchAll('assoc') as $result) {
```

Comment:
```suggestion
        foreach (($statement->fetchAll('assoc') ?: []) as $result) {
```

There is `fetchAssoc()` but not all `fetchAssocAll()` :(

Reply:
Thanks. I didn't consider the scenario where fetchAll() would return false.